recommendations/templates/recommendations/base.html

```diff
@@ -51,6 +50,15 @@ <h1>ReeldIn</h1>
       <div id="searched_movies_container"></div>
     </div>
     <div id="profile_div">
+      <a href="{% url 'recommendations:recommend' %}">
+        {% if user.is_authenticated and recommendations is None %}
+        <button id="get_recommendations_btm">Generate Recommendation</button>
```

Comment:
Not sure if this is from a dev POV (as opposed to a new user) but the "Generate Recommendation" button wasn't so obvious, like it being at the top right where the "Get Recommendation" button was made me instinctively ignore it. I think the button would benefit from having different text and/or different styling as well (ex: dark purple/dark grey) to show that it's a special and different button or something? Idk it just seems kinda off. You could maybe utilize a checkered flag icon to make that more instinctive 'finish' connection but idk if that would look good.

Reply:
i just made it red and i like it? let me know what you think.